### PR TITLE
add tip of w4mjoinpn and w4mcorcov; update w4mclassfilter to tip

### PR DESCRIPTION
**Pull request by eschen42**
I would like to request the following changes:

- Update w4mclassfilter to version at the repository tip
- Add w4mjoinpn and w4mcorcov tools

All of these tools have detailed help and working examples.

I am preparing a manuscript to present the tools.

Bjoern said that there might be suggestions to improve the tools prior to pushing them to UseGalaxy.eu;
I believe that this statement was for my information rather than based on a review of the tools.

### CODE REVIEW

File: metabolomics.yaml.lock [hunk 1]
```diff
@@ -55,7 +55,17 @@ tools:
 - name: w4mclassfilter
   owner: eschen42
   revisions:
-  - 191a720488ce
+  - 1ced8b5dfa3e
```
Comment (bgruening) [inline metabolomics.yaml.lock]: This file is automatically generated. Please do not edit it.

Comment (erasche) [reply metabolomics.yaml.lock]: Rather, please do not remove the old revision, that should continue to be listed. If we need to update a tool that isn't an IUC tool, then that will require a user-committed change to a lock file.

@eschen42 you can use our script to update just the w4m tools in this file:

```
python scripts/update-tool.py --owner eschen42 metabolomics.yaml
```

**Comment by eschen42** [comment]
Helena, That's a great script; it might be good to point to it from the README.md.
I edited metabolomics.yaml.lock as directed.  Thank you.

**Comment by bgruening** [comment]
@eschen42 I have indeed a few request for the tools, sorry. Do you want me to create a PR? I can work on this hopefully next week - does this work for you?

**Comment by eschen42** [comment]
@bgruening I'm afraid that I don't understand the procedure.  If there's more to do beyond this pull request, then, yes, please do what's required or tell me what to do.  Next week will be fine.  Thank you.

**Comment by bgruening** [comment]
@eschen42 please have a look at: https://github.com/HegemanLab/w4mcorcov_galaxy_wrapper/pull/3

**Comment by eschen42** [comment]
I believe that I have addressed the IUC practices concerns raised by @bgruening ; please let me know if other changes are needed.

**Comment by bgruening** [comment]
@eschen42 in this PR are now more commits than needed? Can you rebase against master please?

**Comment by eschen42** [comment]
I couldn't get rebase to work, so I am replacing this PR with [PR 112](https://github.com/usegalaxy-eu/usegalaxy-eu-tools/pull/112)

**Pull request closed without merging**